apps-rendering/src/components/TableOfContents/index.tsx

```diff
@@ -45,38 +40,91 @@ const listStyles: SerializedStyles = css`
 	margin: 0;
 `;
 
-const listItemStyles: SerializedStyles = css`
-	${textSans.xsmall({ fontWeight: 'bold', lineHeight: 'regular' })}
-	border-bottom: 1px solid ${line.primary};
-	padding-top: ${remSpace[2]};
+const defaultListItemStyles = (format: ArticleFormat): SerializedStyles => css`
+	border-top: 1px solid ${border.tableOfContents(format)};
+	padding-bottom: 0;
+
+	&:hover {
+		border-top: 1px solid ${border.tableOfContentsDark(format)};
```

Comment:
The convention we have is to use functions with the "Dark" suffix for dark mode only. It may be preferable to have a specific hover colour for this. Sorry, I know you've already changed this once!